features/analyze.feature

```diff
@@ -115,7 +115,7 @@ Feature: analyze
 
     Scenario: Run the analysis, start with 6 workers.
         Given a directory named "~/.freshli" does not exist
-        When I run `freshli analyze --workers=6 https://github.com/corgibytes/freshli-fixture-java-test`
+        When I run `freshli --workers=6 analyze https://github.com/corgibytes/freshli-fixture-java-test`
```

Comment:
I'd love for us to be able to examine the output to determine if 6 workers have actually started. I think that could be possible by increasing the `loglevel` and then examining the command output to see if there are exactly 6 lines that indicate a worker has started.

(The "exactly 6 lines" approach might not work if don't delay starting the `HangfireBackgroundService` until after the middleware runs, however.)

Reply:
Apparently Hangfire is kind to us and supplies it in one of the first lines as:
```
Worker count: 12
```